contracts/v2-migration/V2ToV3Rollover.sol

```diff
@@ -219,7 +219,7 @@ contract V2ToV3Rollover is IMigration, V2ToV3RolloverBase, FeeLookups {
             opData.newLoanTerms,
             address(this),
             lender,
-            IOriginationController.Signature({
+            IOriginationControllerV3.Signature({
```

Comment:
Thank you for maintaining these files, but going forward we probably can even delete them/archive them in a `v3` branch somewhere

Reply:
Noted, will add this to my list as a clean up task.